File: _querydsl_practice_0329/src/main/java/com/study/springboot/entity/Dept.java
package com.study.springboot.entity;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.Id;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@ToString
public class Dept {
    @Id
    private Integer deptno;
    private String dname;
    private String loc;

}

File: kot0410/src/main/kotlin/com/study/springboot/entity/Dept.kt
package com.study.springboot.entity

import javax.persistence.Entity
import javax.persistence.Id

@Entity
data class Dept(
    @Id val deptno: String, val dname: String, val loc: String
) {
    override fun toString(): String {
        return "Dept(deptno='$deptno', dname='$dname', loc='$loc'"
    }
}